src/jsontestrunner/main.cpp

```diff
@@ -138,7 +138,7 @@ static int parseAndSaveValueTree(const Json::String& input,
         features.allowDroppedNullPlaceholders_;
     builder.settings_["allowNumericKeys"] = features.allowNumericKeys_;
 
-    std::unique_ptr<Json::CharReader> reader(builder.newCharReader());
+    std::unique_ptr<Json::CharReader> reader = builder.makeCharReader();
```

Comment:
Most uses that you're changing don't need to change.

Please make a minimal PR